docs/baseband/gsb/index.rst

```diff
@@ -9,6 +9,224 @@ the `Giant Metrewave Radio Telescope (GMRT) <http://www.gmrt.ncra.tifr.res.in/>`
 backend are found 
 `here <http://gmrt.ncra.tifr.res.in/gmrt_hpage/sub_system/gmrt_gsb/index.htm>`_.
+Its design document is found `on the ArXiv <https://arxiv.org/abs/0910.1517>`_.
+
+.. _gsb_file_structure:
+
+File Structure
+==============
+
+A GSB dataset consists of an ASCII file with a sequence of
+:term:`headers<header>`, and one or more accompanying binary data files.  Each
+line in the header and its corresponding data comprise a :term:`data frame`,
+though these do not have explicit divisions in the data files.
+
+Baseband currently supports two forms of GSB data: **rawdump**, for storing
+real-valued raw voltage timestreams, and **phased**, for storing complex
+pre-channelized data from the GMRT in phased array mode.
+
+Rawdump, (`specification document
+<http://gmrt.ncra.tifr.res.in/gmrt_hpage/sub_system/gmrt_gsb/GSB_rawdump_data_format_v2.pdf>`_)
+has a single data file and accompanying header representing the voltage stream
+from one polarization of a single dish.  The header is just the computer-based
+timestamp (accurate to :math:`1\,\mathrm{\mu s}`)::
+
+    YYYY MM DD HH MM SS.SSSSSS
+
+Rawdump is recorded at a rate of 33.3333... megasamples per second (Msps). 
+Each sample is 4 bits in size, and two samples are grouped into 8-bit words
+such that the oldest sample occupies the least significant bit.  Each data
+frame consists of **4 megabytes** worth, or :math:`2^{23}`, samples; as such,
+the timespan of one frame is exactly **251.658240 ms**.
+
+Data in phased format is spread over multiple binary files and one
+accompanying header file.  The right and left circular polarization data are
+separated into two separate sets of files.  Each polarization is further
+subdivided into multiple files with staggered timestreams such that each file
+contains a portion of every frame.  For example, a typical GSB observation may
+contain four files, two for each polarization.  For each polarization, the
+first half of the payload data of each frame is stored in one file, and the
+second half is stored in the other.
+
+The "raw", or pre-channelized, :term:`sample rate` is either 33.3333... Msps at
+8 bits per sample or 66.6666... Msps at 4 bits per sample (in the latter case,
+sample bit-ordering is the same as for rawdump).  Channelization via fast
+Fourier transform sets the channelized :term:`complete sample` rate to the raw
+rate divided by :math:`2N_\mathrm{F}`, where :math:`N_\mathrm{F}` is the
+number of Fourier channels. The timespan of one frame is **251.658240 ms**, and
+one frame is **8 megabytes** in size, for either raw sample rate.
+
+The phased header's structure is::
+
+    <PC TIME> <GPS TIME> <SEQ NUMBER> <MEM BLOCK>
+
+where ``<PC TIME>`` and ``<GPS TIME>`` are the less accurate computer-based
+and exact GPS-based timestamps, respectively, with the same format as the
+rawdump timestamp; ``<SEQ NUMBER>`` is the frame number; and ``<MEM BLOCK>``
+a redundant modulo-8 shared memory block number not used in analysis.
```

Comment:
delete `not used in analysis` - superfluous with "redundant"

Reply:
Also changed in docstrings.